src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -324,6 +324,13 @@ typedef struct glb_lb_policy {
    * Otherwise, we delegate to the RR policy. */
   size_t serverlist_index;
 
+  bool fallback;
```

Comment:
Makes sense to me. I was thinking it would be too verbose to add more words, but now I feel one word is really vague.